README.md

```diff
@@ -15,12 +15,44 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+

```

Comment:
are these intentional?

Reply:
yeah, I think it makes it easier to skim through